src/Requests/Payload.php

```diff
@@ -66,12 +66,13 @@ protected function generateFromIlluminateRequest(IlluminateRequest $request) : s
     {
         $id = $this->request->headers->get('X-SIGNED-ID', '');
         $timestamp = $this->request->headers->get('X-SIGNED-TIMESTAMP', '');
+        $query = !empty($this->request->query()) ? '?' . http_build_query($this->request->query()) : '';
 
         return json_encode([
             'id' => (string) $id,
             'method' => $this->request->getMethod(),
             'timestamp' => $timestamp,
-            'uri' => (string) $this->request->fullUrl(),
+            'uri' => (string) $this->request->url() . urldecode($query),
```

Comment:
This looks like it produces different URLs that it should. As we discussed, we'll wait for @Jaspaul to get back and have a conversation about it then.

Reply:
@jshayes @Jaspaul @obiwongee Please include me in convo as any changes may required changes to the MSTeams Bot.

Reply:
@crhayes Sure ting my guy